src/components/molecules/UserProfilePicture/UserReactions.scss

```diff
@@ -0,0 +1,115 @@
+@import "scss/constants";
+
+// TODO: should we move these to using CSS --variables rather than these SCSS vars
+$reactionSize: max(2.5vh, 2vw);
+$reactionOffset: calc(-1 * #{$reactionSize} / 2);
+$shoutBounceTranslateXOffset: 50%;
+
+.UserReactions {
+  // Note: we need this to ensure the reactions use this container for their positioning
+  position: relative;
+
+  width: fit-content;
+  height: fit-content;
+
+  &--reaction-left {
+    .UserReactions__reaction {
+      left: $reactionOffset;
+    }
+
+    .UserReactions__shout {
+      right: 50%;
+      transform-origin: right center;
+      animation: shout-bounce--left 5s ease;
```

Comment:
move into a var?

Reply:
Which aspect would you see as being useful to have in a var here out of curiosity?

In testing, I think that the `5s` here corresponds with `REACTION_TIMEOUT` from `src/settings.ts`, and I expect they probably need to be kept in sync. Given that, it might make sense to make this a `--var` and pass it through from the JS side of things.

Reply:
Oh, I think at first I thought they were identical, but then I spotted --left/--right modifiers. I guess time of the reaction can be reused

Reply:
Makes sense. Can probably pull that out into a var (or even a `--var` as mentioned above maybe)

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/1323/commits/10de65755feef5989496b42a13c6b741df68e6d7